lib/experiment/local/fetcher.rb

```diff
@@ -20,7 +20,8 @@ def fetch
       # fetch flag_configs
       headers = {
         'Authorization' => "Api-Key #{@api_key}",
-        'Content-Type' => 'application/json;charset=utf-8'
+        'Content-Type' => 'application/json;charset=utf-8',
+        'X-Amp-Exp-Library' => "experiment-ruby-server/#{VERSION}"
```

Comment:
Do you want to only track the released versions? Currently VERSION is 0.1.0, but there were many beta [releases ](https://github.com/amplitude/experiment-ruby-server/releases) since 0.1.0

Reply:
This will read the actual version including the beta releases.